src/components/Header/Pages.js

```diff
@@ -35,6 +38,7 @@ const playerPages = [{
   name: 'Counts',
 }, {
   name: 'Histograms',
+  extra: histogramNames[0],
```

Comment:
instead of being defined here, shouldn't you set a default when you render the tabs/page?